css/types/counters.json

```
@@ -25,22 +25,22 @@
               "version_added": "8"
             },
             "opera": {
-              "version_added": true
+              "version_added": "10"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "10.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "3.1"
```

I spot checked this and found https://trac.webkit.org/changeset/16721/webkit the changes of which are in the 3.0 release https://trac.webkit.org/browser/webkit/releases/Apple/Safari%203.0/WebCore/css/Counter.h

```suggestion
              "version_added": "3"
```